data-processing/entities/definitions/model/configuration.py

```diff
@@ -15,6 +15,12 @@ class ModelArguments:
             "help": "The model checkpoint for weights initialization. Leave None if you want to train a model from scratch."
         },
     )
+    prediction_type: Optional[str] = field(
+        default=None,
+        metadata={
+            "help": "The model prediction type. One of [TERM-DEF, ABBR-EXP, SYM-NICK]"
```

Comment:
Yes, this is correct, albeit this is only temporary. We are doing this because there is a separate model for each task (which in turn can potentially predict all 3 at once)